File: app/src/main/java/com/pietrantuono/podcasts/subscribedpodcasts/model/SubscribedPodcastModel.java
package com.pietrantuono.podcasts.subscribedpodcasts.model;

import com.pietrantuono.podcasts.addpodcast.model.pojos.Podcast;

import java.util.List;

import rx.Observer;

public interface SubscribedPodcastModel {
    void subscribeToSubscribedPodcasts(Observer<List<Podcast>> observer);
}

File: app/src/main/java/com/pietrantuono/podcasts/subscribedpodcasts/model/SubscribedPodcastModel.kt
package com.pietrantuono.podcasts.subscribedpodcasts.model


import models.pojos.Podcast
import rx.Observer

interface SubscribedPodcastModel {
    fun subscribeToSubscribedPodcasts(observer: Observer<List<Podcast>>)
    fun unsubscribe()
}
